specs.json

```diff
@@ -181,6 +181,11 @@
   "https://w3c.github.io/webappsec-trusted-types/dist/spec/",
   "https://w3c.github.io/webdriver-bidi/",
   "https://w3c.github.io/webrtc-ice/",
+  {
+    "url": "https://webassembly.github.io/exception-handling/js-api/",
+    "shortname": "wasm-js-api-1",
```

Comment:
is it expected we would use the same shortname as a current spec? I was imagining we would mint a special one, while associating it with the series shortname (wasm-js-api).

Reply:
Right, that needs fixing. Tests won't pass once `index.json` gets built because two specs cannot have the same shortname. We'll need to mint a name à la `[base shortname]-fork-[fork-id]` and we may have to relax consistency tests which may yell that the spec's shortname and the series shortname seem mis-aligned.